src/state/reducers/index.js

```diff
@@ -2,3 +2,130 @@
 // This allows for the simplification of flow when importing reducers into your actions throughout your app.
```

Comment:
Not sure if I'm not seeing things correctly, but it appears that you have a 131 line blank file. 
Is this a mistake or is it intentional, for like having space for shoving reducers in at a later date? If it's later, I think we could do with just a few empty lines for now, or maybe even delete the file until it is actually needed.

Reply:
So this file and maybe some other ones were added by a Brad about 2 months ago . I guess it was to be a template for folder structure ?